webpack.config.js

```diff
@@ -22,7 +26,10 @@ module.exports = {
     ),
   },
   output: {
-    path: path.resolve(__dirname, "euphrosyne/assets/dist"),
+    path: path.resolve(
+      dirname(fileURLToPath(import.meta.url)),
```

Comment:
Pour comprendre: quel a été le besoin de changer ça?

Reply:
En gros, j'ai changé la conf pour qu'on utilise les modules ES6 plutôt que CommonJS. __dirname n'a plus était disponible "out of the box"